src/nvim/tui/tui.c

```diff
@@ -1575,7 +1576,7 @@ static void augment_terminfo(TUIData *data, const char *term,
   // can use colons like ISO 8613-6:1994/ITU T.416:1993 says.
   bool has_colon_rgb = false
     // per GNOME bug #685759 and bug #704449
-    || (vte_version >= 3600)
+    || (vte_version >= 3600 && !tmux && !screen)
     || iterm || iterm_pretending_xterm  // per analysis of VT100Terminal.m
     // per http://invisible-island.net/xterm/xterm.log.html#xterm_282
     || true_xterm;
```

Comment:
The check is only for vte because the others depent on what `TERM` is set to.
So you will not need to check if we are for example rxvt because then we can be sure that it didn't detect iterm in that case and so on.